Sources/xcproj/PBXProj.swift

```diff
@@ -111,14 +111,14 @@ final public class PBXProj: Decodable {
         /// It ensures that the generated reference doesn't collide with any existing one.
         ///
         /// - Parameters:
-        ///   - element: object type.
+        ///   - object: the object to generate the reference for.
         ///   - id: object identifier (e.g. path or name)
         /// - Returns: reference.
-        public func generateReference<T: PBXObject>(_ element: T.Type, _ id: String) -> String {
+        public func generateReference<T: PBXObject>(_ object: T, _ id: String) -> String {
```

Comment:
No need for this function to be generic anymore. T can just be a PBXObject

Reply:
Lol @yonaskolb I just updated it before I saw your comment. It's not generic anymore.